.flake8

```diff
@@ -0,0 +1,36 @@
+[flake8]
+show-source = true
+max-line-length = 120
+format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s
+exclude = .git
+application-package-names =
+    march_rqt_gait_selection,
+    march_rqt_input_device,
+    march_rqt_note_taker,
+    march_shared_classes,
+    march_shared_resources
+import-order-style = appnexus
+ignore =
+    # D100: Missing docstring in public module
+    D100,
+    # D102: Missing docstring in public class
+    D101,
+    # D102: Missing docstring in public method
+    D102,
+    # D103: Missing docstring in public function
+    D103,
+    # D104: Missing docstring in public package
+    D104,
+    # D105: Missing docstring in magic method
+    D105,
+    # D106: Missing docstring in public nested class
+    D106,
+    # D107: Missing docstring in __init__
```

Comment:
why add these to the flake8 ignore?

Reply:
They require you to add docstrings to public interfaces. However, that is a lot of docstrings to add and I think it is not always necessary. The name of the function and its parameters could document the method well enough.